news/4.34/jdt.html

```diff
@@ -134,6 +134,22 @@ <h2>Java Views and Dialogs</h2>
 			<img src="images/move-filtersbutton-to-top-other-way.png" alt="old menu">
 		  </p>
       </td>
+	  <tr id="experimental-preference">
+	  	  <td class="title"><strong>Experimental!</strong> Preference added to enable update on runtime behavior</a></td>
+		  <td class="content">
+              <p>
+				This update introduces a new preference setting that allows HiDPI support, enabling monitor-specific UI scaling for improved display clarity. Enabling this preference will also set the <strong>Edge</strong> browser as the default.
```

Comment:
It would be great to explain in one or two sentences what this feature does, so that everyone can understand it. In my opinion, it should at least include:
- Adapt UI scaling to the scaling of the current monitor
- Scaling is performed per window, i.e., multiple windows can have different scaling, depending on the monitor they are placed on